### DISCUSSION TITLE
advertencia de PATH

### DISCUSSION THREAD

**Discussion by venegas55** [LaunchX-InnovaccionVirtual/CursoIntroPython#117] [Q&A] [answered]
que tal compañeros, estoy instalando la actualización de pip y un par de bibliotecas y en CMD siempre me manda esto
![image](https://user-images.githubusercontent.com/89881553/153537938-8b1913e3-4b3c-4763-90db-46dcd6c5975c.png)

¿cómo puedo agregar la ruta a "PATH"?

salu2

**Comment by JairCoronado** [comment]
Puede ser por la version de python si es la mas actual te aparece eso

**Reply by venegas55** [reply]
si es la 3.10 de la tienda de microsoft, hice un tema de agregarlo a path desde propiedades del sistema, veré si eso lo arregla cuando instale otra biblioteca!

**Reply by JairCoronado** [reply]
si quieres que te deje de salir baja al python 3.9 o al 3.7

**Comment by Urielvx** [comment]
Lo que trataste de instalar se instaló correctamente. Antes de que comience el primer "warning" (color amarillo) dice:
... "Running setup.py install for tornado ... done" de color blanco. Por ese lado no hay problema.

Luego, el warning solo indica que los ejecutables (los que tienen .exe, que no creo que los usemos "directamente") están instalados en la ruta o dirección que ahí te muestra. La que empieza con c:\Users\veneg\AppData..., y que consideres pasar ese directorio al PATH. Eso no sé hacerlo pero no creo que te cree problemas. Ojalá alguien pueda resolverlo xD

Ya hasta el final (de cualquier warning amarillo) te da la opción de eliminar el mensaje escribiendo el siguiente comando:
--no-warn-script-location

Yo descargué Python desde la página oficial. Cuando lo estaba instalando me salió la opción de agregarlo al PATH. Podrías desinstalarlo y hacerlo como yo. No está difícil. Solo que es la versión 3.9 la que tengo, creo.

**Reply by venegas55** [reply]
si, el mensaje me queda claro, veo que muchos dicen que usando una versión anterior de python, pero prefiero usar la más reciente y solo agregarlo a path!

**Comment by doguedogue** [comment, marked as answer]
1. Presionas inicio de windows, escribes variables y escoges esa
![image](https://user-images.githubusercontent.com/23409026/153543062-5ef8aee5-9490-483b-916f-2a49c49cf2c6.png)

2. Presionas variables de entorno
![image](https://user-images.githubusercontent.com/23409026/153543138-de176f27-532f-4c99-8b83-ca8dbef5edc0.png)

3.Agregas un path de usuario o en la parte inferio el global
![image](https://user-images.githubusercontent.com/23409026/153543523-75f25d86-3705-4eec-83a8-d19d3dab2337.png)

**Reply by venegas55** [reply]
la ruta en mi computadora era diferente, pero creo que funciona, voy a intentar instalando otro paquete a ver si no me sale la advertencia.
Nota: usé la ruta que viene en la imágen jejeje

**Reply by venegas55** [reply]
📷 si jaló!
gracias! @doguedogue !!
